docs/content/guides/operations/control-plane/metrics/grafana/index.md

```diff
@@ -10,6 +10,10 @@ tags: ["metrics", "observability"]
 
 [Grafana](https://grafana.com/) is an open source visualization and analytics tool that allows you to query, visualize, alert on, and explore your metrics. This guide will show you how to install Grafana and configure it to visualize the Radius control plane metrics from Prometheus.
 
+{{% alert color="primary" %}}
+This how-to article has been validated only on a local Kubernetes cluster.  If you test in other configurations and have questions, please review [Grafana](https://grafana.com/).
+{{% /alert %}}
+
```

Comment:
```suggestion
```

Since the issue is in Prometheus we don't need to list the note on the Grafana guide